nautobot_golden_config/forms.py

```diff
@@ -169,6 +170,8 @@ class ComplianceFeatureForm(
 ):
     """Filter Form for ComplianceFeature instances."""
 
+    slug = SlugField()
```

Comment:
Just validating this won't affect existing slug definitions?

Reply:
This doesn't automatically update the slug field on an object, post update. If a user were to open a feature object in the UI, they will now have the option to regenerate the slug based on the name field input. At this point, this could be updated to a proper slug from the name field, but that is a manual, optional task.